repos/fdbt-types/matchingJsonTypes.ts

```diff
@@ -158,6 +157,7 @@ export interface BaseTicket<T extends TicketType = TicketType> {
     ticketPeriod: TicketPeriod;
     groupDefinition?: GroupDefinition;
     carnet?: boolean;
+    caps?: { id: number };
```

Comment:
This is needed because BaseSchemeOperatorTicket extends BaseTicket.

We need to do what is detailed in the above comment, and also edit BaseTicket to have cap?: DbCap